iree-requirements.txt

```diff
@@ -1,2 +1,3 @@
+--find-links https://iree.dev/pip-release-links.html
 iree-compiler==20240514.893
 iree-runtime==20240514.893
```

Comment:
Not sure how to make this work without either

A) adding `--find-links https://iree.dev/pip-release-links.html` to all install commands for iree-turbine
B) only pinning to versions here that are published to pypi (https://pypi.org/project/iree-compiler/#history)